lib/bootloader_setup.pm

```diff
@@ -1064,6 +1064,11 @@ sub zkvm_add_interface {
     my $mac = get_required_var('VIRSH_MAC');
     # direct access to the tap device, use of $vtap temporarily
     $svirt->add_interface({type => 'direct', source => {dev => $netdev, mode => 'bridge'}, target => {dev => 'macvtap' . $vtap}, mac => {address => $mac}});
+    my $mm_netdev = get_var('MM_NETDEV');
```

Comment:
@rfan1 as you mentioned this fix is for s390x_zkvm, will it need to add the parameter for selecting s390x only?

Reply:
for now it's only needed by s390x multi-machine tests in CC, e.g. netfilter, audit_remote